dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ set(MODEL_FILES
   MinThresholdSwitch.mo
   PulseFixedDuration.mo
   PulseMinimumDuration.mo
+  SectionControl.mo
```

Comment:
About the name. I propose something like DiscreteIntegrator (or something similar, if you have a better proposal. The thing is that it is an integrator that gives an output that's not continuous but discrete). @marcochiaramello a proposal ?

Reply:
Or even DiscreteIntegratorAntiWindUp to be complete !

Reply:
I like the DiscreteIntegratorAntiWindUp proposal